src/components/App/index.test.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
-import App from './App';
-import styles from './App.module.scss';
-import configureStore from './configureStore';
+import App from '.';
+import styles from './styles.module.scss';
+import configureStore from '../../configureStore';
```

Comment:
Maybe it's time to start using absolute paths, if that's an option? All the `../../` in this patch are a bit annoying.